_posts/en/2020-11-17-FWD50-in-the-open.md

```diff
@@ -0,0 +1,69 @@
+- Ann-Marie Cavanagh, Deputy Government Chief Digital Office and Deputy Chief Executive Digital Public Service, New Zealand, shared her governments [Digital Strategy](https://www.digital.govt.nz/digital-government/strategy/strategy-summary/strategy-for-a-digital-public-service/) and [Digital Inclusion Outcomes Framework](https://www.digital.govt.nz/dmsdocument/167~digital-inclusion-outcomes-framework/html) during her talk on COVID-19 and Economic Recovery: New Zealand's lessons learnt on digital inclusion and resilience.
+- The panelists on 'Can the Platform Be Of Service? - The Commitment To Person-centred Design When Adopting Tech Models In The Public Sector', made the highlights of their presentation available in this ['Can the platform be of service' miro board](https://miro.com/app/board/o9J_kg8Sr3c=/).
+
+In a great example of the community working openly, a Google Document was spontaneously created by a FWD50 attendee to capture all the books that were mentioned by speakers. This [list of books](https://docs.google.com/document/d/1Kyq80aMy_a6wnOdaHgzYLqSaqKfmjMmBdDnXKy6N4YQ/edit#heading=h.kr7a4cpsy27s) was shared with all attendees and anyone could add to it when a book was mentioned.
+
+Some articles and other documents/tools that were shared by speakers include:
+
+- Article from the Bennett Institute: [New research identifies effective international cooperation for digital governance](https://www.bennettinstitute.cam.ac.uk/news/new-research-identifies-effective-international-co/)
+- Article from the Harvard Business Review: [Curiosity Is as Important as Intelligence](https://hbr.org/2014/08/curiosity-is-as-important-as-intelligence)
+- Amazon's listing of open code: [Open Government Solutions](https://aws.amazon.com/government-education/government/open-government-solutions/).  (this is not dissimilar to the Government of Canada's [Open Resource Exchange](https://code.open.canada.ca/en/index.html))
+- [Ramy Nasser](https://twitter.com/ramynassar) and [Rebecca Kain](https://twitter.com/rebeccalkain) referenced the [Intelligence Augmentation Design Toolkit](https://futurice.com/ia-design-kit) in their workshop on Creating Resident-Centric AI Solutions Using Design Thinking.
+- The [Pubic Sector Network](https://publicsectornetwork.co/) is a social learning platform that exists to help government around the globe break down silos, collaborate, and work together for better outcomes for citizens.
+
+There were many people who were live-tweeting the event under the [#FWD50](https://twitter.com/search?q=(%23FWD50)%20since%3A2020-11-03&src=typed_query) hashtag. But if going through all of those tweets feels overwhelming, then there were some standout live tweeters who's FWD50 feeds are worth checking out:
+
```

Comment:
Any possibility to embed the tweet directly in the blog post? Instead of the user having to click on each link?

https://help.twitter.com/en/using-twitter/how-to-embed-a-tweet

Reply:
The link itself leads to a query so that they can see all tweets (we can embed specific tweets but OMG there's so many...)

The names we've put below are helping curate the tweets only with #fwd50 hashtags starting from November 1st.

Reply:
More advanced embedded Twitter queries require working with API which then requires a dev account and I feel it's pushing it a bit for this first version :yum: 

Might I suggest a subsequent post linking to juicy content? Adding Sean's blog and others?

Reply:
Oh, Sean's blog is already listed, my bad